shared/test/local_storage.js

```diff
@@ -36,6 +36,16 @@ describe ("Local Storage Test Suite", function() {
     expect(returned["foo"]).toBe("bar");
     expect(returned["bar"]).toBe("bazz");
     expect(returned).toEqual(obj);
-  });
+ });
 
+ it("can set and remove keys from Objects", function() {
+    var obj = {"foo": "bar", "bar": "bazz"};
+    ls.setItem("key", obj);
+    var returned = ls.getItem("key");
+    expect(ls.getItem("key")).toEqual(obj);
+    expect(returned["foo"]).toBe("bar");
+    ls.removeItem("foo");
```

Comment:
Sorry. What I meant is "set and remove Objects". I was trying something else back then. Fixed it. Please review the code.